Assets/_BForBoss/_VFX/Scripts/TransformMover.cs

```diff
@@ -0,0 +1,55 @@
+//-----------------------------------------------------------------------
+// <copyright file="TransformMover.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Collections;
+using System.Collections.Generic;
+using PerigonGames;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace BForBoss
+{
+    public class TransformMover : MonoBehaviour
+    {
+
+        [SerializeField] private Vector3 _point1;       // First point to visit.
```

Comment:
So normally we don't write comments. we write self documenting code. 
Because when people change code, normally they don't touch the comments. And over the years, the comments start to lie. 

So in this case, if i understand the comment correctly. The variable name would be like `_firstPointToVisit`

Reply:
The don't write comments, i picked it up from here
https://www.amazon.ca/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882

If u want to read this, i can send you a little something.

Reply:
Good to know! I'll remove them